### PR TITLE
Hauki 259 global notifications

### PR DESCRIPTION
**Pull request by minevala**
- Render toast-component outside application to make it visible all the time
- Show notification toast for date-period delete operations

### CODE REVIEW

File: src/components/notification/Toast.test.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        ?.parentElement || null;
+
+    act(() => {
+      toast.success(props);
+    });
+
+    expect(document.body.contains(container)).toBe(true);
+
+    act(() => {
+      jest.advanceTimersByTime(20000);
```
Comment (VilleMiekkoja) [inline src/components/notification/Toast.test.tsx:44]: This is a clever way to hide the notification 👍

File: src/components/notification/Toast.tsx [hunk 2]
```diff
@@ -8,51 +9,70 @@ type ToastProps = {
   dataTestId?: string;
 };
 
-const Toast = ({
+const removeContainer = (container: HTMLDivElement): void => {
+  setTimeout(() => {
+    if (container && container.parentElement) {
```
Comment (VilleMiekkoja) [inline src/components/notification/Toast.tsx:14]: This looks familiar 😁

File: src/resource/resource-opening-hours/opening-period/OpeningPeriod.tsx [hunk 3]
```diff
@@ -83,9 +84,22 @@ export default function OpeningPeriod({
...
         <ConfirmationModal
-          onConfirm={(): void => {
+          onConfirm={async (): Promise<void> => {
             if (datePeriod.id) {
-              deletePeriod(datePeriod.id);
+              try {
+                await deletePeriod(datePeriod.id);
+                toast.success({
+                  label: 'Aukiolo poistettu onnistuneesti',
+                  text: `Aukiolo ${datePeriod.id} poistettu onnistuneesti`,
```
Comment (VilleMiekkoja) [inline src/resource/resource-opening-hours/opening-period/OpeningPeriod.tsx:93]: Im not sure if we should show database IDs in toast messages? Are those IDs shown somewhere else? I wonder if people understand what they are 🤔 . Maybe we could show the period title there instead?

**Comment by VilleMiekkoja** [comment]
Nice work and a great feature! 🎖️

File: src/resource/resource-opening-hours/opening-period/OpeningPeriod.tsx (hunk 3 above)
Comment (minevala) [reply src/resource/resource-opening-hours/opening-period/OpeningPeriod.tsx:93]: True, I can change this one

**Review by VilleMiekkoja — APPROVED** [review]

**Review by VilleMiekkoja — APPROVED** [review]

**Pull request merged**